### PR TITLE
rectangle tests

### PR DESCRIPTION
**Pull request by sudakevych-andrii**

### CODE REVIEW

File: tests/tests.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import unittest
+import math
+
+
+from homework import Rectangle
+
+
+class TestRectangle(unittest.TestCase):
+    def arg_list(self):
+        return [[3, 4], [5, 12], [7, 24], [5, 5]]
```
Comment (alchemixXx) [inline tests/tests.py]: it's better to use setUp in tests

File: tests/tests.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        for i in range(len(self.arg_list())):
+            if i == (len(self.arg_list()) - 1):
+                rect = self.rect_with_args(i)
+                result = rect.get_radius_of_inscribed_circle()
+                diagonal = rect.get_rectangle_diagonal()
+                self.assertEqual(result, diagonal / (2 * math.sqrt(2)))
+            else:
+                with self.assertRaises(ValueError):
+                    self.rect_with_args(i).get_radius_of_inscribed_circle()
+
```
Comment (alchemixXx) [inline tests/tests.py]: always split it into separate test: passed and failed

File: tests/tests.py (hunk 1 above)
Comment (sudakevych-andrii) [reply tests/tests.py]: Done

File: tests/tests.py (hunk 2 above)
Comment (sudakevych-andrii) [reply tests/tests.py]: Done

**Pull request merged**